Index: docker/libnetwork/store.go
===================================================================
--- docker.orig/libnetwork/store.go
+++ docker/libnetwork/store.go
@@ -5,7 +5,6 @@ import (
 	"strings"
 
 	"github.com/docker/libkv/store/boltdb"
-	"github.com/docker/libkv/store/consul"
 	"github.com/docker/libkv/store/etcd"
 	"github.com/docker/libkv/store/zookeeper"
 	"github.com/docker/libnetwork/datastore"
@@ -13,7 +12,6 @@ import (
 )
 
 func registerKVStores() {
-	consul.Register()
 	zookeeper.Register()
 	etcd.Register()
 	boltdb.Register()
Index: docker/engine/pkg/discovery/kv/kv.go
===================================================================
--- docker.orig/engine/pkg/discovery/kv/kv.go
+++ docker/engine/pkg/discovery/kv/kv.go
@@ -10,7 +10,6 @@ import (
 	"github.com/docker/go-connections/tlsconfig"
 	"github.com/docker/libkv"
 	"github.com/docker/libkv/store"
-	"github.com/docker/libkv/store/consul"
 	"github.com/docker/libkv/store/etcd"
 	"github.com/docker/libkv/store/zookeeper"
 	"github.com/sirupsen/logrus"
@@ -38,12 +37,10 @@ func init() {
 func Init() {
 	// Register to libkv
 	zookeeper.Register()
-	consul.Register()
 	etcd.Register()
 
 	// Register to internal discovery service
 	discovery.Register("zk", &Discovery{backend: store.ZK})
-	discovery.Register("consul", &Discovery{backend: store.CONSUL})
 	discovery.Register("etcd", &Discovery{backend: store.ETCD})
 }
 
Index: docker/libnetwork/hostdiscovery/hostdiscovery.go
===================================================================
--- docker.orig/libnetwork/hostdiscovery/hostdiscovery.go
+++ docker/libnetwork/hostdiscovery/hostdiscovery.go
@@ -10,7 +10,6 @@ import (
 	"github.com/docker/docker/pkg/discovery"
 	// Including KV
 	_ "github.com/docker/docker/pkg/discovery/kv"
-	"github.com/docker/libkv/store/consul"
 	"github.com/docker/libkv/store/etcd"
 	"github.com/docker/libkv/store/zookeeper"
 	"github.com/docker/libnetwork/types"
@@ -24,7 +23,6 @@ type hostDiscovery struct {
 }
 
 func init() {
-	consul.Register()
 	etcd.Register()
 	zookeeper.Register()
 }
