From: Reinhard Tartler <siretart@tauware.de>
Description: Disable tests that assume we run as root

Index: docker.io/engine/daemon/daemon_linux_test.go
===================================================================
--- docker.io.orig/engine/daemon/daemon_linux_test.go
+++ docker.io/engine/daemon/daemon_linux_test.go
@@ -335,6 +335,7 @@ func TestRootMountCleanup(t *testing.T)
 }
 
 func TestIfaceAddrs(t *testing.T) {
+	t.Skip("Disable test that requires netns")
 	CIDR := func(cidr string) *net.IPNet {
 		t.Helper()
 		nw, err := types.ParseCIDR(cidr)
Index: docker.io/engine/internal/safepath/join_test.go
===================================================================
--- docker.io.orig/engine/internal/safepath/join_test.go
+++ docker.io/engine/internal/safepath/join_test.go
@@ -51,6 +51,7 @@ func TestJoinEscapingSymlink(t *testing.
 }
 
 func TestJoinGoodSymlink(t *testing.T) {
+	t.Skip("Debian-local: Test requires root")
 	tempDir := t.TempDir()
 	dir, err := filepath.EvalSymlinks(tempDir)
 	assert.NilError(t, err, "filepath.EvalSymlinks failed for temporary directory %s", tempDir)
@@ -85,6 +86,7 @@ func TestJoinGoodSymlink(t *testing.T) {
 }
 
 func TestJoinWithSymlinkReplace(t *testing.T) {
+	t.Skip("Debian-local: Test requires root")
 	tempDir := t.TempDir()
 	dir, err := filepath.EvalSymlinks(tempDir)
 	assert.NilError(t, err, "filepath.EvalSymlinks failed for temporary directory %s", tempDir)
@@ -126,6 +128,7 @@ func TestJoinWithSymlinkReplace(t *testi
 }
 
 func TestJoinCloseInvalidates(t *testing.T) {
+	t.Skip("Debian-local: Test requires root")
 	tempDir := t.TempDir()
 	dir, err := filepath.EvalSymlinks(tempDir)
 	assert.NilError(t, err)
Index: docker.io/engine/pkg/archive/archive_test.go
===================================================================
--- docker.io.orig/engine/pkg/archive/archive_test.go
+++ docker.io/engine/pkg/archive/archive_test.go
@@ -1264,6 +1264,7 @@ func TestXGlobalNoParent(t *testing.T) {
 // also verifies that the permissions of explicit directories are respected.
 func TestImpliedDirectoryPermissions(t *testing.T) {
 	skip.If(t, runtime.GOOS == "windows", "skipping test that requires Unix permissions")
+	t.Skip("Debian-local: tests not running as root")
 
 	buf := &bytes.Buffer{}
 	headers := []tar.Header{{
Index: docker.io/engine/hack/test/unit
===================================================================
--- docker.io.orig/engine/hack/test/unit
+++ docker.io/engine/hack/test/unit
@@ -37,6 +37,9 @@ if [ -n "${base_pkg_list}" ]; then
 		${TESTFLAGS} \
 		${base_pkg_list}
 fi
+
+exit 0
+# Debian-local: libnetwork tests invoke iptables, and this are unsuitable for running as part of the package build
 if [ -n "${libnetwork_pkg_list}" ]; then
 	# libnetwork tests invoke iptables, and cannot be run in parallel. Execute
 	# tests within /libnetwork with '-p=1' to run them sequentially. See
